conda/environments/rmm_dev_cuda9.2.yml

```diff
@@ -12,4 +12,5 @@ dependencies:
 - cffi>=1.10.0
 - pytest
 - cudatoolkit=9.2
-- spdlog >=1.4.2
+- spdlog=1.7.0
```

Comment:
We can just remove this file since we no longer support cuda 9.2

Reply:
Wasn't sure if RMM still supported 9.2, or if it was dropped RAPIDS-wide.